content/en/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -0,0 +1,88 @@
+start and remain running during the entire life of the Pod, which is useful for
+running supporting services separated from the main application containers.
+
+If a `readinessProbe` is specified for this init container, its result will be used
+to determine the `ready` state of the Pod.
+
+Since these containers are defined as init containers, they benefit from the same
+ordering and sequential guarantees as other init containers, allowing them to
+be mixed with other init containers into complex Pod initialization flows.
+
+Compared to regular init containers, sidecar-style init containers continue to
+run and the next init container can begin starting once the kubelet has set
+the `started` container status for the sidecar-style init container to true.
+That status either becomes true because there is a process running in the
+container and no startup probe defined, or
+as a result of its `startupProbe` succeeding.
+
+This feature can be used to implement the sidecar container pattern in a more
+robust way, as the kubelet always restarts a sidecar container if it fails.
+
+Here's an example of a Deployment with two containers, one of which is a sidecar:
+
+{{% code_sample language="yaml" file="application/deployment-sidecar.yaml" %}}
+
+This feature is also useful for running Jobs with sidecars, as the sidecar
+container will not prevent the Job from completing after the main container
+has finished.
+
+Here's an example of a Job with two containers, one of which is a sidecar:
+
+{{% code_sample language="yaml" file="application/job/job-sidecar.yaml" %}}
+By default, this feature is not available in Kubernetes. To avail this feature,
+we need to enable the feature gate `SidecarContainers` by setting the value **true**
+to it.
+
+## Resource sharing within containers
+
+Given the ordering and execution for init containers, the following rules
+for resource usage apply:
+
+* The highest of any particular resource request or limit defined on all init
+  containers is the *effective init request/limit*. If any resource has no
+  resource limit specified this is considered as the highest limit.
+* The Pod's *effective request/limit* for a resource is the higher of:
+  * the sum of all app containers request/limit for a resource
+  * the effective init request/limit for a resource
+* Scheduling is done based on effective requests/limits, which means
+  init containers can reserve resources for initialization that are not used
+  during the life of the Pod.
+* The QoS (quality of service) tier of the Pod's *effective QoS tier* is the
+  QoS tier for init containers and app containers alike.
+
+Quota and limits are applied based on the effective Pod request and
+limit.
+
+Pod level control groups (cgroups) are based on the effective Pod request and
+limit, the same as the scheduler.
+
+## {{% heading "whatsnext" %}}
+
+* Read a blog post on [native sidecar containers](https://kubernetes.io/blog/2023/08/25/native-sidecar-containers/).
```

Comment:
Instead of absolute link, we could use relative link here.
```suggestion
* Read a blog post on [native sidecar containers](/blog/2023/08/25/native-sidecar-containers/).
```

Reply:
Good catch!
Updated the same.

Reply:
@T-Lakshmi Please click the "Resolve conversation" button when you fully agreed to the suggestions made by other reviwers AND you have updated the PR accordingly.
Marking conversation resolved saves a lot of time for other reviewers and yourself.
You know what are the discussion topics unsettled, what are the ones that have been resolved. A PR becomes cleaner on the GitHub webiste.